meta-ostro/recipes-extended/libiio/libiio_git.bb

```diff
@@ -0,0 +1,14 @@
+DESCRIPTION = "Analog Devices Libiio"
+DEPENDS = "libxml2 bison flex avahi"
+LICENSE = "LGPLv2+"
```

Comment:
Yes, it is LGPLv2.1, the same as meta-openembedded/meta-oe/recipes-extended/openlmi/openlmi-providers_0.6.0.bb, it is using ""LGPLv2+".